ambry-cloud/src/main/java/com/github/ambry/cloud/HelixVcrClusterParticipant.java

```diff
@@ -191,6 +193,17 @@ public void participate() throws Exception {
     }
     manager.connect();
     helixAdmin = manager.getClusterManagmentTool();
+
```

Comment:
Looks like instance config values will be set everytime a vcr node restarts. It should be enough to set it just once. Would this code be more appropriate for the helix tool that setsup the vcr cluster (HelixBootstrapUpgradeTool or HelixVcrPopulateTool?)

Reply:
Setting it as default properties is the best method, but Helix doesn't support this in FULL_AUTO mode.

I made a change to set configs only when HELIX_ENABLED is false to avoid multiple setups.